helpers/core.js

```diff
@@ -166,6 +164,14 @@ var helpers = {
 				if (asVariable) {
 					aliases[asVariable] = expr[key];
 				}
+				if (!isEmptyObject(hashOptions)) {
+					if (hashOptions.value) {
+						aliases[hashOptions.value] = expr[key];
+					}
+					if (hashOptions.key) {
+						aliases[hashOptions.key] = key;
+					}
```

Comment:
How about `hashOptions.index`?

Reply:
Shouldn't that just be available when iterating over arrays? The docs say "`index`: The index of the current array item"

Reply:
I'm talking about the alias created when using hash expressions. So if you do something like

```
{{#each(items, val=value num=index)}}
```